qutebrowser/misc/editor.py

```diff
@@ -124,6 +124,6 @@ def edit(self, text):
         self._proc.error.connect(self.on_proc_error)
         editor = config.get('general', 'editor')
         executable = editor[0]
-        args = [self._filename if arg == '{}' else arg for arg in editor[1:]]
+        args = [arg.replace('{}', self._filename) if '{}' in arg else arg for arg in editor[1:]]
```

Comment:
Yep, I thought about that, I was trying to change as little as possible, sounds good though. I'll clean this up when I fix the tests. I'll try to get everything green for you within a week or so.

I know it's a small change but it's the cherry on top for me. I'll be gradually moving myself off of Firefox onto qute, I'll contribute where ever I can if I start using it day to do. Glad it's Python!

Tox sounds cool, I was trying to work that out initially, what tooling I needed and what I was supposed to run. I probably need to get virtualenv working again too.